docs/source/author/translations.rst

```diff
@@ -1,4 +1,77 @@
 Translations
 ##############################################################################
 
-Currently translations are not supported.
+Getting Started
+==============================================================================
+
+It's really easy to start contributing translations using the CS Field Guide
+
+1. Read this documentation page, especially the `Translation Notes`_ section.
+2. Create an account on `Crowdin <https://crowdin.com/join>`__.
+3. Visit the `CS Field Guide Crowdin page <https://crowdin.com/project/cs-field-guide>`__.
+4. Start translating!
+
+.. note::
+
+  The in-context translation has been removed temporarily as translation infrastructure is being updated.
+  We aim to `re-add this feature in a future update <https://github.com/uccser/arnold/issues/4>`__.
+
+Crowdin Overview
+==============================================================================
+We use a localisation management platform called `Crowdin <https://crowdin.com/project/cs-field-guide>`__ for translation of CS Field Guide.
+Our project is public, meaning that anyone can create an account and contribute translations.
+
+Crowdin has excellent documentation for translators, and all translators should read the following documents:
+
+- `Crowdin Introduction <https://support.crowdin.com/crowdin-intro/>`_, which provides an overview of the Crowdin platform
+- `Online Editor <https://support.crowdin.com/online-editor/>`_, which explains how to use the Crowdin UI to contribute translations.
+
+Translation Phases
+==============================================================================
+There are 3 phases translation phases used to ensure high quality translation of CS Field Guide content:
```

Comment:
duplicate 'phases'. I suggest:

There are 3 translation phases used to ensure high quality translation of CS Field Guide content: